README.md

```diff
@@ -0,0 +1,5 @@
+# CatPhotoApp
+
+This is the development repo for @FreeCodeCamp 's CatPhotoApp
```

Comment:
~~This is the development repo for @FreeCodeCamp 's CatPhotoApp~~ This is the development repo for [FreeCodeCamp](https://freecodecamp.com)'s CatPhotoApp